servant-server/test/Servant/Server/UsingConfigSpec/CustomCombinator.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- | This is a custom combinator for S.S.UsingConfigSpec. It's split up into
+-- its own module to be able to test how exactly module import work when using
```

Comment:
I wanted to put
- writing the combinator and
- using the combinator

into different modules to be able to look at what imports are needed for what. Mostly I wanted to make sure you don't need to import an internal module for using a combinator with a config.